.github/workflows/pr.yml

```diff
@@ -13,19 +13,10 @@ jobs:
       - name: checkout
         uses: actions/checkout@v2
 
-      - name: make services
-        run: make services
-
       - name: java setup
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
 
-      - name: gradle build
-        run: gradle clean build
-
-      - name: gradle unit
-        run: gradle unit
-
-      - name: gradle integration
-        run: gradle integration-test
+      - name: make integration-all as this includes the unit tests
```

Comment:
Maybe "build and test" would be a better name?

Reply:
done